phys/module_sf_urban.F

```diff
@@ -2647,6 +2756,14 @@ SUBROUTINE urban_var_init(ISURBAN, TSURFACE0_URB,TLAYER0_URB,TDEEP0_URB,IVGTYP,
 !m
 !FS   FRC_URB2D(I,J)=0.
       UTYPE_URB2D(I,J)=0
+
+      distributed_aerodynamics_check: IF (distributed_aerodynamics_option) THEN
+        IF (IVGTYP(I, J) == ISURBAN) THEN
+          UTYPE_URB2D(I, J) = 2
```

Comment:
@epn09  your distributed parameters are only for urban type = 2?

Reply:
@cenlinhe In my understanding, the difference between urban types (low intensity, high intensity, LCZ 1 to 10) in WRF right now are only the differences in the parameters in URBPARM.TBL. When we input a spatial map of these urban parameters from outside, the value of urban type doesn't matter any more. So, it's just set to 2 for no special reasons.

Reply:
OK